### ISSUE TITLE
xliff.jsToXliff12 unexpected parse result with [object Object]

### ISSUE THREAD

**Issue by gjchoi** [locize/xliff#42]
## 🐛 Bug Report

xliff.jsToXliff12 incorrect parsing with source child > [object Object]

## To Reproduce

steps
1. convert xliff to js with xliff.xliff12ToJs command ( this is ok )
2. convert js (result step1) to xliff again with  xliff.jsToXliff12 ( this is unexpected )

### original value
```xliff
      <trans-unit id="ngb.timepicker.AM" datatype="html">
        <source><x id="INTERPOLATION" equiv-text="Step: number;"/></source>
        <context-group purpose="location">
          <context context-type="sourcefile">node_modules/@ng-bootstrap/ng-bootstrap/src/timepicker/timepicker.ts</context>
          <context context-type="linenumber">116,123</context>
        </context-group>
      </trans-unit>
```

> xliff.xliff12ToJs 

```js
"ngb.timepicker.AM": {
      "source": {
        "Standalone": {
          "id": "INTERPOLATION",
          "equiv-text": "Step: number;"
        }
      },
      "additionalAttributes": {
        "datatype": "html"
      }
    }
```

> xliff.jsToXliff12

```xliff
      <trans-unit id="ngb.timepicker.AM" datatype="html">
        <source>[object Object]</source>
      </trans-unit>
```

## Expected behavior

I expect the result as source with <x ..> tag not [object Object]

```xliff
      <trans-unit id="ngb.timepicker.AM" datatype="html">
        <source><x id="INTERPOLATION" equiv-text="Step: number;"/></source>
      </trans-unit>
```

## Your Environment

- *runtime version*: i.e. node v15
- "xliff": "^5.5.3"
- *os*: Mac

**Comment by adrai** [comment]
Would you like to send a Pull Request to address this? Remember to add unit tests.

**Comment by gjchoi** [comment]
@adrai 
hello,
I made a PR about this, but I don't have permission to PR
can you give me a permission?

~~~
Permission to locize/xliff.git denied to gjchoi.
fatal: unable to access 'https://github.com/locize/xliff.git/': The requested URL returned error: 403
~~

**Comment by adrai** [comment]
you need to craeate a fork first: https://jarv.is/notes/how-to-pull-request-fork-github/

**Comment by adrai** [comment]
released with v5.6.0
